modules/cloudevent-trigger/variables.tf

```diff
@@ -23,3 +23,22 @@ variable "private-service" {
     region = string
   })
 }
+
+variable "expiration_policy" {
+  description = "The expiration policy for the subscription."
+  type = object({
+    ttl = optional(string, null)
+  })
+  default = {
+    ttl = "" // This does not expire.
+  }
+}
```

Comment:
Honestly, I struggle to reason about a use case where we would want the subscription itself to expire, which is what this is.  I'd rather omit this than invite the confusion of folks assuming it's a message expiration setting.

Reply:
This probably needs to be `message_retention_duration` instead?

Reply:
I don't think we want that either because AIUI that will apply to messages where delivery has never been attempted.